File: Java/CodingChallenges/LeetCode/RemoveDuplicatesFromSortedArray/Attempt01.java
import java.util.Arrays;

// import jdk.javadoc.internal.doclets.formats.html.SourceToHTMLConverter;

public class Attempt01 {
    public static void main(String[] args) throws Exception {
        int[] nums = { 1, 1, 2 };
        int mySecondInteger = myMethod(nums);
        System.out.print(mySecondInteger);
        
    }

    static int myMethod(int[] parameterArray) {
        System.out.print("\n-------------------------\n");
        System.out.println("Original array: " + Arrays.toString(parameterArray));
        System.out.print("\n-------------------------\n");
        System.out.print("Want to return length of 2 and [1, 2, 2]");
        System.out.print("\n-------------------------\n");


        return 2;
      }
}

File: Languages/JavaScript/CodingChallenges/LeetCode/0509FibonacciNumber/attempt01.js
// attempt01.js

var fib = function (n) {
  if (n === 0) return 0;
  if (n === 1) return 1;
  if (n > 1) return fib(n - 1) + fib(n - 2);
};

// console.log(fib(0));
// console.log(fib(1));
// console.log(fib(2));
// console.log(fib(3));
// console.log(fib(4));
console.log(fib(6));
